pkg/services/images.go

```diff
@@ -287,17 +287,17 @@ func (s *ImageService) CreateRepoForImage(i *models.Image) *models.Repo {
 	}
 	tx := db.DB.Create(repo)
 	db.DB.Save(&repo)
-	fmt.Printf("Repo:: %d\n", repo.ID)
-	fmt.Printf("i.commit:: %d\n", i.Commit.ID)
+	log.Infof("Repo:: %d\n", repo.ID)
+	log.Infof("i.commit:: %d\n", i.Commit.ID)
 	i.Commit.Repo = repo
 	i.Commit.RepoID = &repo.ID
 
 	tx2 := db.DB.Save(i.Commit)
 	if tx2.Error != nil {
-		fmt.Printf("::TX2:: %v\n", tx2.Error)
+		log.Infof("::TX2:: %v\n", tx2.Error)
```

Comment:
```suggestion
		log.Errorf("::TX2:: %v\n", tx2.Error)
```

There's a panic down there and this is our last chance to log this in error level, as far as I can see.